auth-web/src/components/auth/common/AccountBusinessType.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+    <v-container class="view-container">
+        <p class="mb-9" v-if="!govmAccount">
+                {{ $t('accountBusinessTypeText') }}
+        </p>
+        <v-form ref="accountInformationForm" data-test="account-information-form">
+            <v-radio-group
+            row
+            v-model="isBusinessAccount"
+            @change="onOrgBusinessTypeChange"
+            mandatory
+            >
+                <v-row justify="space-between">
+                    <v-col cols="6">
+                        <v-radio
+                        label="Individual Person Name"
+                        :key="false"
+                        :value="false"
+                        data-test="radio-individual-account-type"
+                        ></v-radio>
+                    </v-col>
+                    <v-col cols="6">
+                        <v-radio
+                        label="Business Name"
+                        :key="true"
+                        :value="true"
+                        data-test="radio-business-account-type"
+                        ></v-radio>
+                    </v-col>
+                </v-row>
+            </v-radio-group>
+            <fieldset class="auto-complete-relative account-name" data-test="account-name">
+                <legend class="mb-3"  v-if="govmAccount">Enter Ministry Information for this account</legend>
+                <legend class="mb-3"  v-else-if="!isBusinessAccount">Account Name</legend>
+                <v-slide-y-transition>
+                    <div v-show="errorMessage">
+                    <v-alert type="error" icon="mdi-alert-circle-outline">{{ errorMessage }}</v-alert>
+                    </div>
+                </v-slide-y-transition>
+                <v-text-field
+                filled
+                :label="govmAccount ? 'Ministry Name' : isBusinessAccount ? 'Legal Business Name' : 'Account Name'"
+                v-model.trim="name"
+                :rules="orgNameRules"
+                :disabled="saving || orgNameReadOnly"
+                data-test="input-org-name"
+                :readonly="govmAccount"
+                autocomplete="off"
+                :error-messages="bcolDuplicateNameErrorMessage"
+                v-on:keyup="onOrgNameChange"
+                />
+                <org-name-auto-complete
+                v-if="enableOrgNameAutoComplete && isBusinessAccount"
+                :searchValue="autoCompleteSearchValue"
+                :setAutoCompleteIsActive="autoCompleteIsActive"
+                @auto-complete-value="setAutoCompleteSearchValue">
+                </org-name-auto-complete>
+            </fieldset>
+            <template v-if="govmAccount || isBusinessAccount">
+              <v-expand-transition class="branch-detail" data-test="branch-detail">
+                <v-text-field
+                filled
+                :label="govmAccount ? 'Branch/Division (If applicable)' : 'Branch/Division (If optional)'"
```

Comment:
is the label correct? If optional?

Reply:
It was "optional". After checking with Roy, "If applicable" is fine for both cases. So removing the "optional" wording in label